packages/core/components/Modal/MoveFileManifest/index.tsx

```diff
@@ -0,0 +1,99 @@
+import filesize from "filesize";
+import * as React from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+import { ModalProps } from "..";
+import BaseModal from "../BaseModal";
+import { PrimaryButton } from "../../Buttons";
+import FileDetail from "../../../entity/FileDetail";
+import FileSelection from "../../../entity/FileSelection";
+import { interaction, selection } from "../../../state";
+
+import styles from "./MoveFileManifest.module.css";
+
+/**
+ * Modal overlay for displaying details of selected files for NAS cache operations.
+ */
+export default function MoveFileManifest({ onDismiss }: ModalProps) {
+    const dispatch = useDispatch();
+    const fileService = useSelector(interaction.selectors.getFileService);
+    const fileSelection = useSelector(
+        selection.selectors.getFileSelection,
+        FileSelection.selectionsAreEqual
+    );
+    const moveFileTarget = useSelector(interaction.selectors.getMoveFileTarget);
```

Comment:
Now that we don't have a notion of moving on/off the cache it seems we can remove the idea of having a target. IMO this is likely a YAGNI since other than FMS we won't have control over moving files.

Reply:
resolved in [034549d](https://github.com/AllenInstitute/biofile-finder/pull/311/commits/034549de1962d08622e9c4e1b8ea409741103aa5)